corehq/apps/hqwebapp/static/hqwebapp/js/multiselect_utils.js

```diff
@@ -149,25 +143,32 @@ hqDefine('hqwebapp/js/multiselect_utils', [
     };
 
     /*
-     * A custom binding for setting multiselect properties in knockout content.
+     * A custom binding for setting multiselect properties and additional knockout bindings
      * The only dynamic part of this binding are the options
-     * For a list of configurable properties, see http://loudev.com/ under Options
+     * For a list of configurable multiselect properties, see http://loudev.com/ under Options
      */
     ko.bindingHandlers.multiselect = {
         init: function (element, valueAccessor) {
-            var properties = valueAccessor();
-            multiselect_utils.createFullMultiselectWidget(element, properties);
+            var model = valueAccessor();
+            assertProperties.assert(model, [], ['properties', 'options', 'selectableHeaderTitle', 'selectedHeaderTitle', 'searchItemTitle']);
```

Comment:
Shouldn't this just check for `properties` and `options` now?

Reply:
Yeah this is true. At some point during local testing I hit an assertion issue, but realize that was before I refactored the other multiselect on this page which is what triggered the assertion error. Long story short you are right. I locally tested this just now without the extraneous properties and no assertions are thrown.